src/webGLClient.js

```diff
@@ -340,3 +340,7 @@ WebGLClient.prefetch = function() {
   worker.postMessage({ target: 'gl', op: 'setPrefetched', parameters: parameters, extensions: ctx.getSupportedExtensions(), precisions: precisions, preMain: true });
 };
 
+WebGLClient.postCustomMessage = function(data, options) {
```

Comment:
You're right that it doesn't belong on the `WebGLClient` object. Do you have a preference between a global `function postCustomMessage(...)` or `Module['postCustomMessage'] = ...`?

(My original solution was probably a result of the codebase I work on, which modularizes the generated client code and returns `WebGLClient`, preventing global namespace pollution and allowing multiple instances, but that's my own issue)